assets/queries/cloudFormation/elasticache_with_disabled_at_rest_encryption/query.rego

```diff
@@ -0,0 +1,42 @@
+package Cx
+
+
+CxPolicy [ result  ] { 
+
+  document := input.document[i]
+  resource := document.Resources[key]
+  
+  properties := resource.Properties
+  lower(properties.Engine) == "redis"
+  properties.AtRestEncryptionEnabled == false 
+  result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	    sprintf("Resources.%s.properties", [key]),
```

Comment:
can we refine our searchKey to point to the exact line? By checking the type of resource, we should be able to navigate to this field